cli/args/mod.rs

```diff
@@ -1343,7 +1343,10 @@ fn expand_globs(paths: &[PathBuf]) -> Result<Vec<PathBuf>, AnyError> {
           // true because it copies with sh does—these files are considered "hidden"
           require_literal_leading_dot: true,
         },
-      )?;
+      )
+      .with_context(|| {
+        format!("Failed to expand glob: \"{}\"", escaped_path_str)
```

Comment:
I think we should provide the non-escaped glob?

```suggestion
        format!("Failed to expand glob: \"{}\"", path_str)
```